test/buy-collateral-test.ts

```diff
@@ -543,8 +543,8 @@ describe('buyCollateral', function () {
       expect(normalTotalsBasic.trackingSupplyIndex).to.equal(evilTotalsBasic.trackingSupplyIndex);
       expect(normalTotalsBasic.trackingBorrowIndex).to.equal(evilTotalsBasic.trackingBorrowIndex);
       expect(normalTotalsBasic.totalSupplyBase).to.equal(1e6);
-      // EvilToken attack is using 3000 EVIL tokens, so totalSupplyBase should have 3000e6
-      expect(evilTotalsBasic.totalSupplyBase).to.equal(3000e6);
+      // EvilToken attack should be blocked
+      expect(evilTotalsBasic.totalSupplyBase).to.equal(0);
```

Comment:
Try to expect on error thrown, but somehow seems tricky when the test has pauses Automining and resume after actions are appended 🤔 

Is there a way to catch the error on the failed actions when append during automining is pause, and after automining is resumed?

Reply:
Hmm `expect.to.be.reverted` doesn't work? Does it throw an error?

Reply:
I think only this one doesn't work as it called 
` await ethers.provider.send('evm_setAutomine', [false]);`
then after all transaction step is done, it called 
```
 await ethers.provider.send('evm_mine', [start + 1000]);
 await ethers.provider.send('evm_setAutomine', [true]);
```
To process all steps at once.